src/atlas/functionspace/CubedSphereColumns.cc

```diff
@@ -121,12 +121,7 @@ idx_t CubedSphereColumns<BaseFunctionSpace>::invalid_index() const {
 }
 
 template <typename BaseFunctionSpace>
-idx_t CubedSphereColumns<BaseFunctionSpace>::nb_elems() const {
-    return cubedSphereColumnsHandle_.get()->nb_elems();
-}
-
-template <typename BaseFunctionSpace>
-idx_t CubedSphereColumns<BaseFunctionSpace>::nb_owned_elems() const {
+idx_t CubedSphereColumns<BaseFunctionSpace>::sizeOwned() const {
     return cubedSphereColumnsHandle_.get()->nb_owned_elems();
```

Comment:
Why have internally nb_owned_elems() - would it be tidier to have it all sizeOwned()

Reply:
Good point! I'll do that.

Reply:
Done.